examples/operator/201_customer_tablets.yaml

```diff
@@ -67,16 +103,17 @@ spec:
             mysqld:
               resources:
                 limits:
-                  memory: 256Mi
+                  memory: 512Mi
                 requests:
                   cpu: 100m
-                  memory: 256Mi
+                  memory: 512Mi
             dataVolumeClaimTemplate:
               accessModes: ["ReadWriteOnce"]
               resources:
                 requests:
                   storage: 10Gi
   - name: customer
+    durabilityPolicy: none
     turndownPolicy: Immediate
```

Comment:
I am not running VTOrc for the second keyspace we run. Maybe we should 🤷‍♂️ . I had not added it initially because I didn't want to make the test too bulky.

Reply:
should be fine.